src/reducers/reducer.js

```diff
@@ -12,12 +12,13 @@ import {
   DEFAULT_TOKEN,
   PUSH_API_STATUS,
   FEATURE_TOGGLE_DEFAULTS,
-  PRE_SETTINGS_MAINNET,
   NETWORKSETTINGS_STATUS,
   NANOCONTRACT_REGISTER_STATUS,
-  WALLETCONNECT_NEW_NANOCONTRACT_TX_STATUS,
+  REOWN_NEW_NANOCONTRACT_TX_STATUS,
   NANOCONTRACT_BLUEPRINTINFO_STATUS,
-  WALLETCONNECT_CREATE_TOKEN_STATUS
+  PRE_SETTINGS_NANO_TESTNET,
```

Comment:
question: Linting says this `PRE_SETTINGS_NANO_TESTNET` is never used. Is that a bug or can we remove this constant?

Reply:
Removed!